runtime/jvm/src/test/kotlin/kotlinx/serialization/SerializeRecTest.kt

```diff
@@ -25,6 +25,7 @@ data class Container(
         val data: Data
 )
 
+@Ignore
```

Comment:
It is not clear for me what it tests and what value does it carry.
It started to fail after changes in 66193db0ad338609f3c691923415caa1ea375e48 which actually **fixes** previously incorrect behaviour.

Reply:
It tests order of function calls in double dispatch serializer<>encoder. Because we've fixed behavior, step 0 became an `encodeSerializableValue` instead of `beginStructure`. We can re-enumerate the steps, or call `Container::class.serializer().serialize` directly in the line 32.